### PR TITLE
save tabs width in vertical and horizontal terminal panel positions

### LINKED ISSUE #121833: Keep track of user's tab view width in both orientations
Repro:

1. Resize tabs view, remember size
2. Move panel to right
3. Resize tabs view, remember that size too
4. Move panel to bottom, it should match the size in step 1
5. Move panel to right, it should match the size in step 3

### PR DESCRIPTION
**Pull request by meganrogge**
This PR fixes #121833

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts [hunk 1]
```diff
@@ -172,9 +172,11 @@ export class TerminalTabbedView extends Disposable {
 	}
 
 	private _getLastWidgetWidth(): number {
-		const storedValue = this._storageService.get(TABS_WIDGET_WIDTH_KEY, StorageScope.WORKSPACE);
+		const widthKey = this._panelOrientation === Orientation.VERTICAL ? TABS_WIDGET_WIDTH_VERTICAL_KEY : TABS_WIDGET_WIDTH_HORIZONTAL_KEY;
+		const storedValue = this._storageService.get(widthKey, StorageScope.WORKSPACE);
+
 		if (!storedValue || !parseInt(storedValue)) {
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts:178]: ensures the min width is used for vertical

File: src/vs/workbench/contrib/terminal/browser/terminalGroup.ts [hunk 2]
```diff
@@ -470,9 +471,7 @@ export class TerminalGroup extends Disposable implements ITerminalGroup {
 				this.resizePanes(this._initialRelativeSizes);
 				this._initialRelativeSizes = undefined;
 			}
-			if (terminalPositionChanged && this._splitPaneContainer.orientation === Orientation.VERTICAL) {
-				this._onPanelMovedToSide.fire();
-			}
+			this._onPanelOrientationChanged.fire(this._splitPaneContainer.orientation);
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminal/browser/terminalGroup.ts:474]: Think we want to keep this wrapped in `if (terminalPositionChanged)`

File: src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts (hunk 1 above)
Comment (Tyriar) [reply src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts:178]: Let's add a comment explaining

**Review by Tyriar — APPROVED** [review]
Some minor comments

**Pull request merged**